app/models/app.rb

```diff
@@ -32,6 +34,21 @@ def linked_service(service)
     linked_services.find_by!(service: service)
   end
 
+  def fetch_letsencrypt_status_from_server!
+    letsencrypt_email = SshExecution.new(server).
```

Comment:
Is this something you want to do in a model? It's quite an expensive operations which might live better somewhere else (i.e. a service object or something like that, or at least make it not attached to the actual `App` model)

Reply:
Yep that sounds good. Since we don't have such a layer in our app yet, I could simply create a job for now and call that synchronously.